src/Sarif.Multitool/MultitoolFileHelpers.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using Microsoft.CodeAnalysis.Sarif.Readers;
+using Newtonsoft.Json;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    /// <summary>
+    /// Helper functions for the multitool.
+    /// </summary>
+    static class MultitoolFileHelpers
+    {
+        public static SarifLog ReadSarifFile(string filePath)
```

Comment:
>ReadSarifFile [](start = 31, length = 13)

we could provide an extension method or direct "LoadFrom" method where a SarifLog hydrates itself based on a file path.